sshfs.c

```diff
@@ -3494,6 +3596,15 @@ static int read_password(void)
 		perror("Failed to allocate locked page for password");
 		return -1;
 	}
+#ifdef __APPLE__
```

Comment:
Agreed, so done. I guess this is only because MacFUSE wanted to protect everything under `__APPLE__` to keep their changes in the fork isolated and easier to look for.